lib/domain-page/ensService.ts

```diff
@@ -101,13 +106,39 @@ export const getENSDomainDataThroughSubgraph = async (
   return data;
 };
 
+function getParent(domain: string): string {
+  const parts = domain.split(".");
+
+  // remove the first part and return the remaining domain
+  return parts.slice(1).join(".");
+}
+
+const getBasicENSDomainData = async (name: string): Promise<DomainData> => {
+  const domainAdd = await getResolver(publicClient, { name });
+  const domainOwner = await getOwner(publicClient, { name });
+  const wrapperData = await getWrapperData(publicClient, { name });
+
+  return {
+    owner: domainOwner?.owner ?? "0x",
+    parent: getParent(name),
+    subdomains: [],
+    subdomainCount: 0,
+    resolver: {
+      id: "",
+      address: domainAdd as Address,
+      texts: {},
+      addresses: [],
+    },
+    expiryDate: wrapperData?.expiry?.date?.getTime()!,
```

Comment:
```suggestion
    expiryDate: wrapperData?.expiry?.date?.getTime() || undefined,
```

Reply:
`getBasicENSDomainData` should throw an error if  `wrapperData?.expiry?.date?.getTime()` is not defined